src/components/List.js

```diff
@@ -11,6 +11,13 @@ const List = ({ results, setSearchTerm, searchTerm, token }) => {
   function checkTime(time) {
     return Date.now() - time <= 86400000; //number of milliseconds equal to 24 hours
   }
+  function handleDelete(result) {
+    let response = window.confirm(
+      `Permanently remove "${result.name}" from your shopping list? \nYou cannot undo this action, and this item's purchase history will be lost.`,
```

Comment:
One kind of cool thing about template literals like this, is you don't need to use the `\n`.  You can just format it the way you'd like to show it. https://www.javascripttutorial.net/es6/javascript-template-literals/#:~:text=The%20template%20literals%20make%20multiline,line%20up%20with%20proper%20indentation.

Reply:
Oh cool! I didn't know that!  I had some trouble with the spacing when I tried just doing a return.  If I want to keep the second line tabbed in the code, it adds space in the alert...
<img width="546" alt="Screen Shot 2020-09-02 at 10 39 19 PM" src="https://user-images.githubusercontent.com/50968795/92065251-561a3e00-ed6d-11ea-9726-613693d72ef4.png">
<img width="461" alt="Screen Shot 2020-09-02 at 10 39 12 PM" src="https://user-images.githubusercontent.com/50968795/92065257-59152e80-ed6d-11ea-84ba-1cf2fe109516.png">

Reply:
I think you have to _not_ indent it like we'd expect, but not sure without trying it out. Either way works, though, so nbd.

Reply:
Removed it!